teams/clippy-contributors.toml

```diff
@@ -0,0 +1,24 @@
+name = "clippy-contributors"
+subteam-of = "clippy"
+
+[people]
+leads = []
+members = [
+    "GuillaumeGomez",
+    "J-ZhengLi",
+    "m-rph",
+    "pitaj",
+]
+alumni = []
+
+[[github]]
+orgs = ["rust-lang"]
+
+[website]
+name = "Clippy-Contributors"
+description = "Regular contributors to the Clippy linter"
+zulip-stream = "clippy"
+repo = "https://github.com/rust-lang/rust-clippy"
+
+[[zulip-groups]]
+name = "T-clippy"
```

Comment:
Should this be

```suggestion
name = "T-clippy-contributors"
```

Reply:
I think it makes sense to include them in the normal `T-clippy` Zulip group. We get very few pings, like one every two weeks when we have a team meeting, where the contributors are also invited to.

Even if another team would ping us for a design decision, the feedback from them would be useful. That's why I set it to `T-clippy`. Though, I'm open to change this :)

Reply:
Then you need to add `clippy-contributors` to the Zulip group definition in clippy.toml using the `extra-teams` key. Right now, adding a duplicate team definition causes the last definition to be processed to override all other definitions which is obviously a bug. I would argue we need to throw a validation error in this case instead of allowing the group definitions be split across files.

Reply:
In that case, I would go with Ryan's original suggestion and just add them to `T-clippy-contributors`. No need to ping them for things they don't necessarily have to pay attention to.